src/resources/application.properties

```diff
@@ -9,3 +9,7 @@ logging.level.root=info
 logging.level.org.springframework.web=info
 logging.file.path=.
 logging.type=file
+#Ignite configuration
+#ignite.host=localhost
```

Comment:
This is a very good start with the abstraction layer on DB access. 

Couple of suggestions: 
- We likely want to uncomment the ignite configuration. 
- Place ignite cluster on other hosts, and support remote IPs/URLs instead of localhost
- Deploy Ignite clusters as a K8s service

Reply:
I think the remaining item is to update the one-box deployment script as well as the k8s yaml file, respectively.

https://github.com/futurewei-cloud/alcor/blob/master/scripts/deploy.sh
https://github.com/futurewei-cloud/alcor/tree/master/kubernetes

How did you test the change locally? deploy a Ignite container and verify locally?

Reply:
Let us address this item in next PR.